frontend/test/metabase/scenarios/collections/bookmarks.cy.spec.js

```diff
@@ -0,0 +1,92 @@
+import _ from "underscore";
+import { restore, sidebar } from "__support__/e2e/cypress";
+
+describe("Bookmarks in a collection page", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("cannot add bookmark to root collection", () => {
+    cy.visit("/collection/root");
+
+    cy.icon("bookmark").should("not.exist");
+  });
+
+  it("can add and remove bookmarks from collection from its page", () => {
+    cy.visit("/collection/1");
+
+    // Add bookmark
+    cy.icon("bookmark").click();
+
+    // Bookmark Icon should be blue once Collection is bookmarked
+    cy.icon("bookmark")
+      .should("have.css", "color")
+      .and("eq", "rgb(80, 158, 227)");
+
+    sidebar().within(() => {
+      cy.findByText("Bookmarks");
+      cy.findByText("Bobby Tables's Personal Collection");
```

Comment:
Hard coding this could backfire in the future. To make it more resilient, let's import ADMIN from `cypress_data.js` and then construct this string using admin's first and last name, please.

Reply:
Also, it might make sense to use `data-testid` for individual bookmarks in the sidebar (or even unordered list / table which would allow us to use `byRole` selectors).

Reply:
> or even unordered list / table

Agree that this would be better.

There's some necessary consistency with the collection links in the sidebar which may open Pandora's box, later with styling and so on.

Thinking about moving forward, are you ok with leaving the html tags as they are?

Reply:
Yup. Leave them.